examples/darwin-framework-tool/commands/common/CHIPToolKeypair.mm

```diff
@@ -79,7 +79,12 @@ - (SecKeyRef)publicKey
         };
         _mPublicKey = SecKeyCreateWithData((__bridge CFDataRef) publicKeyNSData, (__bridge CFDictionaryRef) attributes, nullptr);
     }
-    return _mPublicKey;
+
+    if (_mPublicKey) {
+        return CFRetain(_mPublicKey);
```

Comment:
```suggestion
        CFRetain(_mPublicKey);
        return _mPublicKey;
```
avoids complaint from compiler when building `darwin-framework-tool`: `error: cannot initialize return object of type 'SecKeyRef' (aka '__SecKey *') with an rvalue of type 'CFTypeRef' (aka 'const void *')`